frontend/src/components/common/ModalBottomSheet.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from "react"
+
+import styled from "styled-components"
+
+import FeatherIcon from "feather-icons-react"
+import { Sheet } from "react-modal-sheet"
+
+interface BottomSheetProps {
+    onClose: () => void
+    initialSnap?: number
+    title?: string | null
+    icon?: React.ReactNode | null
+    handleBack?: (() => void) | null
+    children: React.ReactNode
+}
+
+const ModalBottomSheet = ({
+    onClose,
+    initialSnap = 0,
+    title = null,
+    icon = null,
+    handleBack = null,
```

Comment:
```suggestion
    title,
    icon,
    handleBack,
```
이 셋은 옵셔널 필드로, 값이 없으면 `undefined`입니다. 이 셋 prop은 `undefined`여도 문제가 없어보입니다.
`handleBack`의 경우에는 이벤트 핸들러는 보통 함수가 없으면 `undefined`를 전달합니다.